### PR TITLE
Create Resolves # 1815 NOTES - FOSTER CARE HCAPPvbs

### PR DESCRIPTION
**Pull request by devonne01**
Blip; This is a case note template for when a HCAPP is received for a foster care client.

### CODE REVIEW

File: Script Files/NOTES/Resolves # 1815 NOTES - FOSTER CARE HCAPPvbs [hunk 1]
```diff
@@ -0,0 +1,121 @@
+'STATS GATHERING----------------------------------------------------------------------------------------------------
+name_of_script = "NOTES - COUNTY BURIAL APPLICATION.vbs"
```
Comment (theVKC) [inline Script Files/NOTES/Resolves # 1815 NOTES - FOSTER CARE HCAPPvbs]: This needs to be updated...

File: Script Files/NOTES/Resolves # 1815 NOTES - FOSTER CARE HCAPPvbs [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+		text_from_the_other_script = fso_command.ReadAll
+		fso_command.Close
+		Execute text_from_the_other_script
+	END IF
+END IF
+'END FUNCTIONS LIBRARY BLOCK================================================================================================
+
+'Required for statistical purposes==========================================================================================
+STATS_counter = 1               'sets the stats counter at one
+STATS_manualtime = 500          'manual run time in seconds
```
Comment (theVKC) [inline Script Files/NOTES/Resolves # 1815 NOTES - FOSTER CARE HCAPPvbs]: This seems high...

File: Script Files/NOTES/Resolves # 1815 NOTES - FOSTER CARE HCAPPvbs [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+
+'THE SCRIPT--------------------------------------------------------------------------------------------------------------------------------------
+'connecting to BlueZone, and grabbing the case number
+EMConnect ""
+Call MAXIS_case_number_finder(case_number)
+
+'calling the dialog---------------------------------------------------------------------------------------------------------------
+DO
+	Dialog County_Burial_Application_Received
```
Comment (theVKC) [inline Script Files/NOTES/Resolves # 1815 NOTES - FOSTER CARE HCAPPvbs]: This is the wrong dialog name...

**Comment by theVKC** [comment]
@devonne01: I'm not able to approve this. The title of the script is invalid (it's listed as "RESOLVES #" on the title and that would never work). Can you update the title and address/fix the comments I added into the script, and then I'll approve this.

**Comment by devonne01** [comment]
I'm on it, I will work on it this morning and get back to you when I'm done.

**Comment by theVKC** [comment]
Great! No worries. Thanks for your help @devonne01!

**Comment by theVKC** [comment]
@devonne01, you and Kelly just did the same thing.

image
 
Need that "resolves #1815" out of the name of your script file. Other than that, this looks much better. :)

**Comment by theVKC** [comment]
OK @devonne01: it looks perfect except you have ONE STRAY SPACE before the "NOTES - ..." part of your title. Unfortunately, it's throwing an error. Can you clear that space out for me? Grrr, I wish it weren't this finicky!

**Pull request merged**